### ISSUE TITLE
DBdiagram embed error

### ISSUE THREAD

**Issue by huketo** [outline/outline#7281]
## DBdiagram embed error

DBdiagram embed link is changed
=> https://dbdiagram.io/e/652cb7fdffbf5169f0bd10c9/669bd2f88b4bb5230ee060dd

![image](https://github.com/user-attachments/assets/8c6cf17c-7644-47c3-bcb6-793d9030eb36)

```tsx
new EmbedDescriptor({
    title: "DBDiagram",
    keywords: "diagrams database",
    regexMatch: [new RegExp("^https://dbdiagram.io/(embed|d)/(\\w+)$")],
    transformMatch: (matches) => `https://dbdiagram.io/embed/${matches[2]}`,
    icon: <Img src="/images/dbdiagram.png" alt="DBDiagram" />,
  }),
```
[shared/editor/embeds/index.tsx](https://github.com/outline/outline/blob/7849af6887951e825d2b3f30132f73f2b5b6897b/shared/editor/embeds/index.tsx#L208C3-L214C6)

#3124 

## My Solution

Change embed link `https://dbdiagram.io/e/652cb7fdffbf5169f0bd10c9/669bd2f88b4bb5230ee060dd` => `https://dbdiagram.io/embed/652cb7fdffbf5169f0bd10c9`

![image](https://github.com/user-attachments/assets/6044c499-b2ca-4404-ab6f-53ad522d84e7)

It Work 😎

![image](https://github.com/user-attachments/assets/fd62cc12-7c00-4ff3-925a-9a374823e2b4)

**Comment by jesili** [comment]
Hi, i found Canvas link does't work. The information is invalid embedded link

**Comment by tommoor** [comment]
What is a canvas link? What format does it take? Are you running the latest version that includes this fix?

**Comment by jesili** [comment]
> What is a canvas link? What format does it take? Are you running the latest version that includes this fix?

Sorry, it's Canva. The link looks like `https://www.canva.com/design/DAGM_7Yg8RM/SzRj2KScCxexL6k7f_wqww/view`, I'm running 0.78.0 in a selfhost server.

**Comment by jesili** [comment]
I also tried mindmeister, the link looks like `https://mm.tt/app/map/3391581462?t=pZJbFRnUZt`. It seems doesn't match the current version.

**Comment by tommoor** [comment]
Canva link detection just fixed here d2f5ac3d5301d292ddf9dd3e2b1e3f4a411e6af2
For Mindmeister that link is not embeddable, maybe try the embed iframe code.
